packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import {
+  BaseComponent,
   EventGroup,
```

Comment:
Can you please run `rush change` and add a change request to patch, with comments:

DetailsList: setting initial focus on a row is now done asynchronously to improve initial paint time by avoiding an unncessary layout.